WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -229,10 +233,53 @@ private extension PaymentCaptureOrchestrator {
                                  currency: order.currency,
                                  receiptDescription: receiptDescription(orderNumber: order.number),
                                  statementDescription: statementDescriptor,
+                                 receiptEmail: provideReceiptEmailIfNecessary(from: order),
                                  paymentMethodTypes: paymentMethodTypes,
                                  metadata: metadata)
     }
 
+    private func synchronizePlugins(from siteID: Int64) async throws {
+        return try await withCheckedThrowingContinuation({
+                (continuation: CheckedContinuation<(), Error>) in
+            let systemPluginsAction = SystemStatusAction.synchronizeSystemPlugins(siteID: siteID) { result in
+                if case let .failure(error) = result {
+                    DDLogError("[PaymentCaptureOrchestrator] Error syncing system plugins: \(error)")
+                    continuation.resume(throwing: error)
+                } else {
+                    continuation.resume(returning: ())
+                }
+            }
+            ServiceLocator.stores.dispatch(systemPluginsAction)
+            })
+    }
+
+    /// We do not need to set the receipt email if WCPay is installed and active
+    /// and its version is higher or equal than 4.0.0, as it does itself in that case.
+    private func provideReceiptEmailIfNecessary(from order: Order) -> String? {
+        let paymentsPluginsDataProvider = PaymentsPluginsDataProvider()
+
+        let wcPay = paymentsPluginsDataProvider.getWCPayPlugin()
+        let stripe = paymentsPluginsDataProvider.getStripePlugin()
+
+        guard !paymentsPluginsDataProvider.bothPluginsInstalledAndActive(wcPay: wcPay, stripe: stripe) else {
+            // This case should not happen, shall we fatal error here?
```

Comment:
My 1ct - yes better to fatal, because this means that something really wrong goes in the code and the feature won't work anyway